tests/unit/backend/corpora/api_server/curator/collection/test_curation_collections.py

```diff
@@ -302,6 +319,23 @@ def test__get_nonexistent_collection__Not_Found(self):
         res = self.app.get("/curation/v1/collections/test_collection_id_nonexistent")
         self.assertEqual(404, res.status_code)
 
+    def test__get_tombstoned_collection__Not_Found(self):
+        tombstoned_collection = self.generate_collection(
+            self.session, tombstone=True, name="tombstoned collection", visibility=CollectionVisibility.PUBLIC
+        )
+        self.generate_dataset(self.session, collection_id=tombstoned_collection.id, tombstone=True)
+        res = self.app.get(f"/curation/v1/collections/{tombstoned_collection.id}")
+        self.assertEqual(404, res.status_code)
+
+    def test__get_collection_without_tombstoned_datasets__OK(self):
```

Comment:
```suggestion
    def test__get_collection_with_tombstoned_datasets__OK(self):
```

Reply:
Ah, yeah. That actually makes a bit more sense.